src/bertconf_bert_loader.erl

```diff
@@ -5,7 +5,7 @@
          code_change/3, terminate/2]).
 
 -include("bertconf.hrl").
--record(state, {watches, changes=[], old_tables=[], reloader=undefined}).
+-record(state, {ref, watches, changes=[], old_tables=[], reloader=undefined, needs_reload=false}).
```

Comment:
Maybe this could be timer_ref? This wasn't really clear until I saw how the state record is built.